browser/ui/brave_layout_constants.cc

```diff
@@ -62,7 +62,7 @@ std::optional<int> GetBraveLayoutConstant(LayoutConstant constant) {
     case LOCATION_BAR_CHILD_CORNER_RADIUS:
       return 4;
     case TAB_SEPARATOR_HEIGHT: {
-      return 24;
+      return 16;
```

Comment:
Not sure if we need to return the old value if `HorizontalTabsUpdateEnabled()` is `false`. Hopefully that flag will go away very soon anyway 😄

Reply:
tbh, I think it still looks fine :laughing: 
![image](https://github.com/user-attachments/assets/78fa54f0-750e-4a8d-bd25-d65bc5f32e8f)

Reply:
Thanks for checking! Looks good to me 👍